deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -683,7 +683,7 @@ spec:
       #      - -c
       #      - >-
       #        mkdir -p /etc/elastic-agent/inputs.d &&
-      #        wget -O - https://github.com/elastic/elastic-agent/archive/8.10.tar.gz | tar xz -C /etc/elastic-agent/inputs.d --strip=5 "elastic-agent-8.10/deploy/kubernetes/elastic-agent-standalone/templates.d"
+      #        wget -O - https://github.com/elastic/elastic-agent/archive/main.tar.gz | tar xz -C /etc/elastic-agent/inputs.d --strip=5 "elastic-agent-main/deploy/kubernetes/elastic-agent-standalone/templates.d"
```

Comment:
This seems to be pulling the version from here

https://github.com/elastic/elastic-agent/blob/4c591212077ff2ef5730bdb0cbb43c0551e58307/deploy/kubernetes/Makefile#L4

Which seems to be because the PR in https://github.com/elastic/elastic-agent/commit/eb99faedfbde5e1dbe2f3f9339272153fd074b41#diff-dd2f89350a6f317b4477133b711f5a91cfb2b1b2dbce1350912d9e1ef833365d only updated the stack-version and not the docs-version.

Reply:
I would revert this and change docs-branch to 8.10 https://github.com/elastic/elastic-agent/blob/4c591212077ff2ef5730bdb0cbb43c0551e58307/version/docs/version.asciidoc#L2

Reply:
Actually Beats leaves it as main, so 🤷 on which one is right. Nobody has ever complained about this.

https://github.com/elastic/beats/pull/36342

Reply:
Actually I suspect that BRANCH_VERSION and BEAT_VERSION should come from the same source and the Makefile is wrong.

https://github.com/elastic/elastic-agent/blob/4c591212077ff2ef5730bdb0cbb43c0551e58307/deploy/kubernetes/Makefile#L3-L4

Reply:
Both should be using `:stack-version:`

Reply:
hmmm, okay. Weird this is just failing now.